docs/developer-guide/ksqldb-rest-api/streaming-endpoint.md

```diff
@@ -77,6 +77,25 @@ won't contain newlines.
 ]
 ```
 
+Starting from 0.18, variable substitution can be applied by passing a map of variables and
+definitions to the `sessionVariables` argument of the request body and referencing variables by
+enclosing them in `${}`. For example, the following requests are equivalent:
+
+```json
+{
+"sql": "select concat(A, '${fruit}') from foo", 

```

Comment:
Can we make this example more realistic to avoid user confusion? A user seeing this might think this is a real query (that should be accepted by the ksqlDB server), which it's not.